packages/xchain-cosmos-sdk/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@xchainjs/xchain-cosmos-sdk",
+  "version": "0.1.0",
+  "description": "Genereic Cosmos SDK client for XChainJS",
+  "keywords": [
+    "XChain",
+    "CosmosSDK"
+  ],
+  "author": "XChainJS",
+  "homepage": "https://github.com/xchainjs/xchainjs-lib",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "module": "lib/index.esm.js",
+  "typings": "lib/index.d.ts",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:xchainjs/xchainjs-lib.git"
+  },
+  "scripts": {
+    "clean": "rimraf lib/**",
+    "build": "yarn clean && rollup -c",
+    "test": "jest --passWithNoTests",
```

Comment:
add `  "e2e": "jest --config jest.config.e2e.js",` for terminal support.

Reply:
Done